slacker/__init__.py

```diff
@@ -47,6 +46,7 @@ def __init__(self, body):
         self.body = json.loads(body)
         self.successful = self.body['ok']
         self.error = self.body.get('error')
+        self.message = self.body.get('response_metadata', {}).get('messages')
```

Comment:
Could you remove this line as it's not a part of this PR? There's [another PR](https://github.com/os/slacker/pull/142) addressing this.